requirements.yml

```diff
@@ -4,3 +4,23 @@
   name: openshift-contrib
 
 - src: ansible.kubernetes-modules
+
+- src: https://github.com/openstack/openstack-ansible-plugins
```

Comment:
Nice.  All these will need to be added to the kubevirt-apb's requirements.yml.

Reply:
Will do.